spawns.lua

```diff
@@ -125,18 +125,65 @@ NorthGeorgiaFARPTransportSpawns = {
     ["SE"] = {Spawner("SE FARP HELO"),nil, SEFARPLogiSpawn},
     ["MK"] = {Spawner("MK FARP HELO"), nil, MaykopLogiSpawn}
 }
+
+-- Tells a tanker (or other aircraft for that matter) to orbit between two points at a set altitude
+
+BladderPos = {}
```

Comment:
What is "Bladder"? I'm reading it as the human organ but can't figure out what else it could mean.

Reply:
It's a flexible cell used to hold liquid (like the organ) - the classname I made up from Jester's "Stratobladder" designation for the KC aircraft.

Although since this specific function is pretty generic, I might rename/remove it and just name the functions a bit nicer.

Reply:
OK cool, the liquid holder (same function as the human organ, heh) is what I was thinking of. I think renaming it could be better. Can't think of one I like yet, but I'll let you know if I do and we still need one.